app/views/returns/vatReturnDetails.scala.html

```diff
@@ -173,8 +180,17 @@ <h3 class="bold-small form-group">@messages("yourVatReturn.additionalInfo")</h3>
     </section>
 
     <div class="form-group">
-        @expandingHelpSection(messages("yourVatReturn.helpTitle"), "vatReturn", "vatReturn", hiddenHelpTextContent)
+        <button class="btn button"
+        onclick="javascript:window.print()">@messages("yourVatReturn.print")
+        </button>
     </div>
 
+    @if(appConfig.features.enablePrintPastReturns()) {
+        <div class="form-group">
+        @expandingHelpSection(messages("yourVatReturn.helpTitle"), "vatReturn", "vatReturn", hiddenHelpTextContent)
+        </div>
+
+        <div class="onlyprint foot">@messages("yourVatReturn.footer", s"${currentDate.toLongDate}")</div>
```

Comment:
Hey sam , it seems that feature switch is removing the link on the page instead of the button ,
Could you please change this so the button is inside the logic for the feature switch 
Thanks